crates/duty-tracker/src/contract_state_machine.rs

```diff
@@ -291,12 +294,18 @@ pub enum OperatorDuty {
     PublishRootNonce {
         /// Transaction ID of the DRT
         deposit_request_txid: Txid,
+
+        /// The schnorr key that locks the funds in the DRT for the user to take back after a
+        /// parameterized refund delay.
         takeback_key: XOnlyPublicKey,
     },
 
     /// Instructs us to send out signatures for the deposit transaction.
     PublishRootSignature {
+        /// The nonces received from peers.
```

Comment:
```suggestion
        /// The public nonces received from peers.
```
just for the sake of completeness.